src/index.html

```diff
@@ -22,6 +22,370 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="index.html"
+        class="header__logo"
+      >
+        <img
+          src="../src/images/logo.png"
+          alt="logo"
```

Comment:
Ensure that image paths are correct. The `src` attribute usually points to the `images` folder without `../src/`, unless the project structure explicitly requires such a path.